### DISCUSSION TITLE
Création personnage

### DISCUSSION THREAD

**Discussion by KevinJ-Dev** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#312] [Q&A] [answered]
Bonsoir,

J'aurais besoin d'aide au niveau de la création du personnage et je pense que ceci à déjà été remonter,
Lors de la création du personnage je ne vois pas ou et comment avoir la boite de dialogue pour rentrer les informations du personnage, telle que, le nom, prénom, date naissance,

![11](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/54015f36-2963-4473-b8cb-6d31a0d086fc)
![111](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/3a2642dc-df37-4f66-9ff8-325589992a36)
![1111](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/ed474fda-9315-4b99-b829-c6ded590bc4c)
![11111](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/c13a0c02-2535-411f-b047-1abaf88ae055)

je peu mettre le nom prénom seulement en mode admin, dans la gestion du personnage "créer un nouveau personnage"
![2](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/d5b3e372-e013-4e48-99bf-38a32c1200ab)
![22](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/082e025b-705c-44b6-8c9d-c022e2a76da7)

Merci de votre aide

**Comment by Laikker** [comment]
Ce type d'information passe par le site web de SOZ qui n'est pas open source.
A toi d'adapter

**Reply by KevinJ-Dev** [reply, marked as answer]
Ok merci bien
